server/controllers/users.js

```diff
@@ -37,7 +43,19 @@ class UserController {
       await user.createProfile();
 
       const token = await Token.create(payload);
-      return res.status(201).json({ status: 'success', message: 'User created successfully', user: userExtractor(user, token) });
+
+      const tokenExpiry = Date.now() + ((Number(process.env.RESET_TOKEN_EXPIRE)) || 75600000);
+      const verifyToken = randomstring.generate(40);
+
+      const verifyDetails = {
+        verifyToken, tokenExpiry, userId: user.id
+      };
+
+      await VerifyUser.create({ ...verifyDetails });
+      sendVerifyMailToken(verifyToken, user.email, user.username);
+      return res.status(201).json({
+        status: 'success', message: 'User created successfully', user: userExtractor(user, token), verifyToken
```

Comment:
why exactly are you sending the `verifyToken` ?

Reply:
For my test to have access to the verification token.